diff --git a/node_modules/expo-router/build/native-tabs/NativeBottomTabs/NativeTabsView.js b/node_modules/expo-router/build/native-tabs/NativeBottomTabs/NativeTabsView.js
index 9feb57f..a9ae0e5 100644
--- a/node_modules/expo-router/build/native-tabs/NativeBottomTabs/NativeTabsView.js
+++ b/node_modules/expo-router/build/native-tabs/NativeBottomTabs/NativeTabsView.js
@@ -153,7 +153,12 @@ function convertOptionsIconToPropsIcon(icon) {
         return undefined;
     }
     if ('sf' in icon && icon.sf) {
-        return { sfSymbolName: icon.sf };
+        return {
+            ios: {
+                type: 'sfSymbol',
+                name: icon.sf
+            }
+        };
     }
     else if ('src' in icon && icon.src) {
         return { templateSource: icon.src };